### DISCUSSION TITLE
why can't I see comments?

### DISCUSSION THREAD

**Discussion by hub2git** [ReVanced/revanced-discussions#933] [Question] [answered]
![Screenshot_20230310-232831](https://user-images.githubusercontent.com/7141051/224471373-b8aac895-e31d-4b9f-a377-3987b9705710.png)
![Screenshot_20230310-232930](https://user-images.githubusercontent.com/7141051/224471409-514ff867-93f3-4c2d-a337-2353cd7647e7.png)

![Screenshot_20230310-233009](https://user-images.githubusercontent.com/7141051/224471505-250a73c7-4443-4d19-94ac-ba12b4740ec5.png)

**Comment by oSumAtrIX** [comment, marked as answer]
Reset app data

**Reply by Fffjjy** [reply]
P

**Comment by hub2git** [comment]
Thank you. That fixed it. Perhaps we can add this to the Instructions or
FAQ?

**Reply by oSumAtrIX** [reply]
You are the only one with that issue, it is not a frequent issue

**Comment by sarkarsupankar126-commits** [comment]
> ![Screenshot_20230310-232831](https://user-images.githubusercontent.com/7141051/224471373-b8aac895-e31d-4b9f-a377-3987b9705710.png) ![Screenshot_20230310-232930](https://user-images.githubusercontent.com/7141051/224471409-514ff867-93f3-4c2d-a337-2353cd7647e7.png)
> 
> ![Screenshot_20230310-233009](https://user-images.githubusercontent.com/7141051/224471505-250a73c7-4443-4d19-94ac-ba12b4740ec5.png)

![Screenshot_20230310-232831](https://user-images.githubusercontent.com/7141051/224471373-b8aac895-e31d-4b9f-a377-3987b9705710.png)
![Screenshot_20230310-232930](https://user-images.githubusercontent.com/7141051/224471409-514ff867-93f3-4c2d-a337-2353cd7647e7.png)


![Screenshot_20230310-233009](https://user-images.githubusercontent.com/7141051/224471505-250a73c7-4443-4d19-94ac-ba12b4740ec5.png)

**Comment by sarkarsupankar126-commits** [comment]
😟🥺

**Comment by sarkarsupankar126-commits** [comment]
![Uploading IMG_20260501_200938754.jpg…]()

**Comment by sarkarsupankar126-commits** [comment]
So had dinner
